src/azul/service/elasticsearch_service.py

```diff
@@ -587,6 +587,12 @@ def transform_request(self,
                 es_response = es_search.execute(ignore_cache=True)
             except elasticsearch.NotFoundError as e:
                 raise IndexNotFoundError(e.info["error"]["index"])
+            except elasticsearch.RequestError:
```

Comment:
This needs to be more specific to the type of error we're expecting in the case of an empty index.

Reply:
I might be misunderstanding the context in which error type is expressed here but `RequestError` is the type of error for an empty index.